cfgov/cfgov/settings/local.py

```diff
@@ -23,7 +23,7 @@
     'disable_existing_loggers': False,
     'handlers': {
         'console': {
-            'level': 'INFO',
+            'level': 'DEBUG',
```

Comment:
This change shouldn't affect how things are logged anywhere else because [we define the fallback logger](https://github.com/cfpb/cfgov-refresh/blob/541656522efca8570d3ab7355a5da246dc521a38/cfgov/cfgov/settings/local.py#L33) with a level of `INFO`. That should filter out any `logger.debug()` messages unless the console handler is explicitly invoked.